### PR TITLE
Add animation CSS properties compat data

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds compat data for the following CSS properties: 

* [animation-delay](https://developer.mozilla.org/docs/Web/CSS/animation-delay)
* [animation-direction](https://developer.mozilla.org/docs/Web/CSS/animation-direction)
* [animation-duration](https://developer.mozilla.org/docs/Web/CSS/animation-duration)
* [animation-fill-mode](https://developer.mozilla.org/docs/Web/CSS/animation-fill-mode)
* [animation-iteration-count](https://developer.mozilla.org/docs/Web/CSS/animation-iteration-count)
* [animation-name](https://developer.mozilla.org/docs/Web/CSS/animation-name)
* [animation-play-state](https://developer.mozilla.org/docs/Web/CSS/animation-play-state)
* [animation-timing-function](https://developer.mozilla.org/docs/Web/CSS/animation-timing-function)
* [animation](https://developer.mozilla.org/docs/Web/CSS/animation)

### CODE REVIEW

**Comment by ddbeck** [comment]
OK, I've applied the changes for the webkit prefixes on Firefox (plus, since it was basically the same change, I fixed up some other places where I had written notes instead of a `flag` statement). Ready for your review, @teoli2003. 😄

File: css/properties/animation-play-state.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            "opera": [
+              {
+                "version_added": "12.10"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "12"
+              }
+            ],
+            "safari": {
```
Comment (teoli2003) [inline css/properties/animation-play-state.json]: safari also supports the unprefixed version

File: css/properties/animation-play-state.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+              "prefix": "-webkit-",
+              "version_added": true
+            }
```
Comment (teoli2003) [inline css/properties/animation-play-state.json]: safari_ios is missing

File: css/properties/animation-timing-function.json [hunk 3]
```diff
@@ -0,0 +1,152 @@
+{
+  "css": {
+    "properties": {
+      "animation-timing-function": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/animation-timing-function",
+          "support": {
+            "webview_android": {
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: Also supports unprefixed version

File: css/properties/animation-timing-function.json [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: nit: -moz- after -webkit-

File: css/properties/animation-timing-function.json [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+                "version_added": "44",
+                "prefix": "-webkit-",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: ie_mobile is missing

File: css/properties/animation-timing-function.json [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+              "version_added": "10"
+            },
+            "opera": [
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: "opera": [
{
"version_added": "12.1",
"version_removed": "15"
},
{
"prefix": "-webkit-",
"version_added": "15"
},
{
"prefix": "-o-",
"version_added": "12",
"version_removed": "15"
}
],

File: css/properties/animation-timing-function.json [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+              {
+                "prefix": "-o-",
+                "version_added": "12"
+              }
+            ],
+            "opera_android": {
+              "prefix": "-o-",
+              "version_added": "12"
+            },
+            "safari": {
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: safari also supports the unprefixed version

File: css/properties/animation-timing-function.json [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+            "opera": [
+              {
+                "version_added": "12.10"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "12"
+              }
+            ],
+            "opera_android": {
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: "opera": [
{
"version_added": "12.1",
"version_removed": "15"
},
{
"prefix": "-webkit-",
"version_added": "15"
},
{
"prefix": "-o-",
"version_added": "12",
"version_removed": "15"
}
],

File: css/properties/animation-timing-function.json [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false,
+                "notes": "The name of the <code>frames()</code> timing function is <a href='https://github.com/w3c/csswg-drafts/issues/1301'>under discussion</a>, so it is disabled in browser release versions until a final decision is reached. It is enabled in Nightly/Canary only."
+              },
+              "opera_android": {
+                "version_added": true
+              }
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: status for frames() is missing: experimental and on standard track.

File: css/properties/animation-timing-function.json [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false,
+                "notes": "The name of the <code>frames()</code> timing function is <a href='https://github.com/w3c/csswg-drafts/issues/1301'>under discussion</a>, so it is disabled in browser release versions until a final decision is reached. It is enabled in Nightly/Canary only."
+              },
+              "opera_android": {
+                "version_added": true
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: This is susipicious. Any source for it?

File: css/properties/animation-timing-function.json [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+        "frames": {
+          "__compat": {
+            "description": "frames()",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "The name of the <code>frames()</code> timing function is <a href='https://github.com/w3c/csswg-drafts/issues/1301'>under discussion</a>, so it is disabled in browser release versions until a final decision is reached. It is enabled in Nightly/Canary only."
+              },
+              "chrome_android": {
+                "version_added": true
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: This is susipicious. Any source for it?

File: css/properties/animation-timing-function.json [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: nit: -moz- after -webkit-

File: css/properties/animation-timing-function.json [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+              "version_added": "12"
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "prefix": "-webkit-",
+              "version_added": true
+            }
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: safari also supports the unprefixed version

File: css/properties/animation.json [hunk 14]
```diff
@@ -0,0 +1,149 @@
+{
+  "css": {
+    "properties": {
+      "animation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/animation",
+          "support": {
+            "webview_android": {
```
Comment (teoli2003) [inline css/properties/animation.json]: Unprefixed version supported.
Also I would remove the partial_implementation, the note is enough.

File: css/properties/animation.json [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+            "chrome": [
+              {
+                "version_added": "43"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "chrome_android": {
```
Comment (teoli2003) [inline css/properties/animation.json]: Chrome_android does support it (see other properties)

File: css/properties/animation-timing-function.json [hunk 16]
```diff
@@ -0,0 +1,152 @@
...
+            },
+            "chrome": [
+              {
+                "version_added": "43"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: chrome_android is missing

File: css/properties/animation.json [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "frames_timing_function": {
+          "__compat": {
+            "description": "<code>frames</code> timing function",
+            "support": {
```
Comment (teoli2003) [inline css/properties/animation.json]: Add other mobile versions, with false. (ie and ie_mobile must have false there too)

File: css/properties/animation.json [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": false,
+                "notes": "The name of the <code>frames()</code> timing function is <a href='https://github.com/w3c/csswg-drafts/issues/1301'>under discussion</a>, so it is disabled in browser release versions until a final decision is reached. It is enabled in Nightly/Canary only."
+              }
+            }
```
Comment (teoli2003) [inline css/properties/animation.json]: status is missing: experimental/standard_track

File: css/properties/animation.json [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+                "version_added": "12"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": true,
+                "version_removed": "12.50",
+                "notes": "See the <a href='http://my.opera.com/ODIN/blog/2012/08/03/a-hot-opera-12-50-summer-time-snapshot'>Opera 12.50 release notes</a>."
+              }
+            ],
+            "safari": {
```
Comment (teoli2003) [inline css/properties/animation.json]: also supports unprefixed version

File: css/properties/animation.json [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+              "prefix": "-webkit-",
+              "version_added": "4"
+            },
+            "safari_ios": {
```
Comment (teoli2003) [inline css/properties/animation.json]: also supports unprefixed version

File: css/properties/animation.json [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
```
Comment (teoli2003) [inline css/properties/animation.json]: nit: -moz- entry after -webkit- entries

File: css/properties/animation-delay.json [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "16"
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply css/properties/animation-delay.json]: It is fine!

File: css/properties/animation-direction.json [hunk 23]
```diff
@@ -0,0 +1,238 @@
...
+                "version_added": "16"
+              },
+              "firefox_android": {
+                "version_added": "16"
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "ie_mobile": {
+                "version_added": null
```
Comment (teoli2003) [inline css/properties/animation-direction.json]: I likely forgot this one: should be true.

File: css/properties/animation-duration.json [hunk 24]
```diff
@@ -0,0 +1,151 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": false
```
Comment (teoli2003) [inline css/properties/animation-duration.json]: true (i likely missed it in the first pass, sorry)

File: css/properties/animation-iteration-count.json [hunk 25]
```diff
@@ -0,0 +1,161 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/animation-iteration-count.json]: true (I likely missed this one during the first pass)

File: css/properties/animation-name.json [hunk 26]
```diff
@@ -0,0 +1,161 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/animation-name.json]: true

File: css/properties/animation-play-state.json [hunk 27]
```diff
@@ -0,0 +1,161 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/animation-play-state.json]: true

File: css/properties/animation-timing-function.json [hunk 28]
```diff
@@ -0,0 +1,201 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/properties/animation-timing-function.json]: true

File: css/properties/animation-timing-function.json (hunk 11 above)
Comment (teoli2003) [reply css/properties/animation-timing-function.json]: Yes, It is incorrect: false
(See: https://www.chromestatus.com/feature/5189363944128512 )

File: css/properties/animation-timing-function.json (hunk 10 above)
Comment (teoli2003) [reply css/properties/animation-timing-function.json]: If not in Chrome/Chromium not in Opera: false

**Comment by ddbeck** [comment]
OK, I've fixed the latest round of comments. Let me know if you want to see any more changes. Also, that Chrome Status site is new to me—a useful resource, thank you!

**Pull request merged**